lib/src/Uncertainty/Algorithm/Optimization/EfficientGlobalOptimization.cxx

```diff
@@ -92,8 +93,9 @@ class ExpectedImprovementEvaluation : public EvaluationImplementation
     const Scalar sk2 = metaModelResult_.getConditionalMarginalVariance(x);
     const Scalar sk = sqrt(sk2);
     if (!SpecFunc::IsNormal(sk)) return Point(1, -SpecFunc::MaxScalar);
-    const Scalar ratio = fmMk / sk;
-    Scalar ei = fmMk * DistFunc::pNormal(ratio) + sk * DistFunc::pNormal(ratio);
+    const Point ratio (1, fmMk / sk);
+    const Normal gaussian(1);
+    Scalar ei = fmMk * gaussian.computeCDF(ratio) + sk * gaussian.computePDF(ratio);
```

Comment:
This is due to 6cde2f5220271f0095a016a21fe81cec2b85d220
My bad ☹️
We need to avoid creating Normal(1). I will implement `DistFunc::dNormal`

Reply:
we need a bench, I guess in 1-d it's not so bad
else we could declare it as static so the ctor is called once